### PR TITLE
Reorder awards to be on right of score

### PR DESCRIPTION
**Pull request by onebytegone**

### CODE REVIEW

**Comment by pbredenberg** [comment]
I tried this at first, but had some problems when the player had both a panda and a raven. Have you tested that situation @onebytegone ?

**Comment by onebytegone** [comment]
Maybe I'm missing something, but it seems to be working... 

demo

File: src/app/leaderboard/leaderboard.component.scss [hunk 1]
```diff
@@ -51,7 +51,8 @@
             color: rgb(7, 34, 8);
             position: relative;
             &_rank {
-                padding-left: 5%;
+                padding-right: 5%;
+                text-align: right;
```
Comment (onebytegone) [inline src/app/leaderboard/leaderboard.component.scss]: Guess this is technically a different change 🤷‍♂️

**Comment by pbredenberg** [comment]
It could be fine, but just to be safe try it with points in the millions. If it looks good, I think we're good to go!

**Comment by onebytegone** [comment]
big-scores

**Pull request merged**